x-pack/solutions/security/plugins/security_solution/server/endpoint/services/workflow_insights/builders/incompatible_antivirus.test.ts

```diff
@@ -154,6 +158,45 @@ describe('buildIncompatibleAntivirusWorkflowInsights', () => {
     });
     const params = generateParams('test.com');
 
+    params.esClient.search = jest.fn().mockResolvedValue({
+      hits: {
```

Comment:
can a type (something like `DeepPartial<SearchHit<SignatureSomething>>`) be added to these object literals, so if a key changes somewhere, `type_check` will find it faster than the tests?

(asking because just come across some outdated stuff in mocks/tests, because they weren't typed, and the tests were still passing)